sphinx_gallery/backreferences.py

```diff
@@ -265,7 +265,7 @@ def identify_names(script_blocks, global_variables=None, node=''):
 .. only:: html
 
   .. image:: /{thumbnail}
-    :alt: {title}
```

Comment:
I might have missed this but why are we removing the alt?

Reply:
Since the current alt is simply the gallery item's title, #998 suggested to remove it because redundant information will be spoken when using a screen reader :slightly_smiling_face:

Reply:
It would be nice to check the title is still read out?